product.html

```diff
@@ -0,0 +1,361 @@
+          <h2 class="services__heading">Track your project performance</h2>
+          <p class="services__text">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua. At vero eos et accusam et justo duo
+            dolores et ea rebum. Stet clita kasd gubergren.
+          </p>
+        </div>
+        <div class="services__body">
+          <div class="services__img-wrapper">
+            <img src="./assets/service2.png" class="services__img" alt="service">
+          </div>
+        </div>
+      </div>
+      <div class="services__row">
+        <div class="services__header">
+          <h4 class="services__label">Lead Generation</h4>
+          <h2 class="services__heading">More leads that convert</h2>
+          <p class="services__text">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua. At vero eos et accusam et justo duo
+            dolores et ea rebum. Stet clita kasd gubergren.
+          </p>
+        </div>
+        <div class="services__body">
+          <div class="services__blur-gradient services__blur-gradient--bottom"></div>
+          <div class="services__img-wrapper">
+            <img src="./assets/stats.png" class="services__img" alt="service">
+          </div>
+        </div>
+      </div>
+      <div class="services__row services__row--reverse">
+        <div class="services__header">
+          <h4 class="services__label">Remote teams</h4>
+          <h2 class="services__heading">Real-time collaboration</h2>
+          <p class="services__text">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+            erat, sed diam voluptua. At vero eos et accusam et justo duo
+            dolores et ea rebum. Stet clita kasd gubergren.
+          </p>
+        </div>
+        <div class="services__body">
+          <div class="services__img-wrapper">
+            <img src="./assets/service2.png" class="services__img" alt="service">
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="features">
+      <div class="features__header">
+        <h2 class="features__heading">Features</h2>
+      </div>
+      <div class="features__body features__body--secondary">
+        <ul class="features__list">
+          <li class="features__item">
+            <div class="features__card features__card--divider">
+              <img src="./assets/encrypt.svg" class="features__card-img" alt="shield">
+              <h4 class="features__card-heading--lg">
+                High security to protect from piracy
+              </h4>
+              <p class="features__card features__card-text--dark-gray">
```

Comment:
Also here, I expect `features__card-text--dark-gray` to have `features__card-text` as an element